botocore/__init__.py

```diff
@@ -51,6 +51,9 @@ def emit(self, record):
 
 BOTOCORE_ROOT = os.path.dirname(os.path.abspath(__file__))
 
+# Used to specify anonymous (unsigned) request signature
+UNSIGNED = ''
```

Comment:
We've been using an sentinel value of `object()` for these values.  It allows us to use identity checks instead of equality checks to avoid certain types of errors.  I'd prefer if we kept to that convention unless there's a strong reason not to.